python/tchannel/tornado/util.py

```diff
@@ -52,3 +53,15 @@ def print_arg(request, index):
                 print chunk
     except Exception:
         pass
+
+
+def deserialize_json(content):
+    if content is None:
+        return ""
```

Comment:
the result of deserializing None is not an empty string. It's an error. 

assert that content is not None